index.html

```diff
@@ -0,0 +1,70 @@
+<html>
+
+
+
+<main id="main">
+  
+    <h1 id="title"> Bombay Bicycle Club</h1>
+    <link rel="stylesheet" href="style.css">
+    <p>4 men Indie ensamble hailing from London</p>
```

Comment:
[OPTIONAL] Please consider using a heading tag here like h4 - h6 since you have already used h1-h3, this makes your code cleaner and more syntactically informative for the visually impaired.